### PR TITLE
ci: refine terraform scripts for debugging and add missing dmsetup/device-mapper in some distros

### PR DESCRIPTION
**Pull request by yangchiu**
#### Which issue(s) this PR fixes:

Issue N/A

#### What this PR does / why we need it:

refine terraform scripts for debugging and add missing dmsetup/device-mapper in some distros

### CODE REVIEW

**Comment by yangchiu** [comment]
Using `vfio_pci` causes Longhorn unable to detect the node or make the node as a `Down` node? More testing needed.

**Comment by derekbit** [comment]
> Using `vfio_pci` causes Longhorn unable to detect the node or make the node as a `Down` node? More testing needed.

Sounds weird.
Can you elaborate more on `unable to detect the node`?

**Comment by derekbit** [comment]
cc @roger-ryao @chriscchien

**Comment by yangchiu** [comment]
> > Using `vfio_pci` causes Longhorn unable to detect the node or make the node as a `Down` node? More testing needed.
> 
> Sounds weird. Can you elaborate more on `unable to detect the node`?

Currently unable to reproduce. Probably AWS issue. Logs has been inserted for debugging: https://github.com/longhorn/longhorn-tests/pull/2043/files#diff-c1fd1979fa9da2add5bdcc4548097abe8e3e7be94060865b06567a14e07e978eR131

File: test_framework/terraform/aws/oracle/main.tf [hunk 1]
```diff
@@ -85,6 +85,14 @@ resource "aws_security_group" "lh_aws_secgrp_controlplane" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  ingress {
+    description = "Allow longhorn-ui nodeport"
+    from_port   = 30000
+    to_port     = 30000
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```
Comment (derekbit) [inline test_framework/terraform/aws/oracle/main.tf:94]: Move the codes out of the commit `ci: enable both uio_pci_generic and vfio_pci` and create a separate commit for the change.

Comment (yangchiu) [reply test_framework/terraform/aws/oracle/main.tf:94]: Moved `vfio` related code to https://github.com/longhorn/longhorn-tests/pull/2056. Thank you!

**Review by roger-ryao — APPROVED** [review]
LGTM

**Pull request merged**